gammapy/modeling/models/temporal.py

```diff
@@ -418,5 +470,112 @@ def from_dict(cls, data):
         return cls.read(data["filename"])
 
     def to_dict(self, full_output=False):
-        """Create dict for YAML serilisation"""
+        """Create dict for YAML serialisation"""
         return {"type": self.tag[0], "filename": self.filename}
+
+
+class PowerLawTemporalModel(TemporalModel):
+    r"""Temporal model with a Power Law decay.
+
+    .. math::
+            F(t) = ((t - t_ref)/t0)^alpha
+
```

Comment:
Add forward reference to example plot, like `For more information see :ref:`powerlaw-temporal-model`.`